arm-relay/2017-04-01/swagger/relay.json

```diff
@@ -0,0 +1,2203 @@
+              "type": "string",
+              "description": "The time the namespace was updated."
+            },
+            "listenerCount": {
+              "readOnly": true,
+              "format": "int32",
+              "type": "integer",
+              "minimum": 0,
+              "maximum": 25,
+              "description": "The number of listeners for this relay. min :1 and max:25 supported"
+            },
+            "relayType": {
+              "type": "string",
+              "description": "WCFRelay Type.",
+              "enum": [
+                "NetTcp",
+                "Http"
+              ],
+              "x-ms-enum": {
+                "name": "Relaytype",
+                "modelAsString": false
+              }
+            },
+            "requiresClientAuthorization": {
+              "type": "boolean",
+              "description": "true if client authorization is needed for this relay; otherwise, false."
+            },
+            "requiresTransportSecurity": {
+              "type": "boolean",
+              "description": "true if transport security is needed for this relay; otherwise, false."
+            },
+            "userMetadata": {
+              "type": "string",
+              "description": "usermetadata is a placeholder to store user-defined string data for the HybridConnection endpoint.e.g. it can be used to store  descriptive data, such as list of teams and their contact information also user-defined configuration settings can be stored."
+            }
+          },
+          "description": "Properties of the WcfRelay Properties."
+        }
+      },
+      "description": "Description of WcfRelays Resource.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ]
+    },
+    "RelayNamespaceListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RelayNamespace"
+          },
+          "description": "Result of the List Namespace operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of Namespaces"
+        }
+      },
+      "description": "The response of the List Namespace operation."
+    },
+    "RelayNamespace": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Sku of the Namespace."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RelayNamespaceProperties",
+          "description": "Description of Relay Namespace"
+        }
+      },
+      "description": "Description of a Namespace resource.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "RelayNamespaceProperties": {
+      "properties": {
+        "provisioningState": {
```

Comment:
should it be an an enum, with modelAsString:true ? see https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-enum